Dockerfile.aro-multistage

```
@@ -0,0 +1,22 @@
+# Uses a multi-stage container build to build the RP.
+#
+# TODO:
+# Currently the docker version on our RHEL7 VMSS uses a version which
+# does not support multi-stage builds.  This is a temporary stop-gap
+# until we get podman working without issue
+FROM registry.access.redhat.com/ubi7/go-toolset:1.14.12 AS builder
```

```suggestion
FROM registry.access.redhat.com/ubi7/go-toolset:1.14 AS builder
```

^ consider the above so that if they roll a new 1.14.x we get it automatically

---

Updated